dockerfiles/lib/src/internal/dir/che-dir.ts

```diff
@@ -201,8 +208,16 @@ export class CheDir {
       this.fs.statSync(this.cheFile);
       // we have a file
     } catch (e) {
-      Log.getLogger().debug('No chefile defined, use default settings');
-      return;
+
+      // if there is a .Chefile, use it, else return
+      try {
+        this.fs.statSync(this.dotCheFile);
+        Log.getLogger().debug('The alternate file .Chefile is present at ', this.dotCheFile);
+        this.cheFile = this.dotCheFile;
+      } catch (e) {
+        Log.getLogger().debug('No chefile defined, use default settings');
```

Comment:
maybe this error message can be improved now:
- chefile --> Chefile
- and maybe add the possibility to use Chefile: "No Chefile or .Chefile defined, use default settings"